feathr_project/feathr/anchor.py

```diff
@@ -33,16 +39,9 @@ def to_feature_config(self) -> str:
                     {% endfor %}
                 }
             }
-        }
-        
-        {% if not source.name == "PASSTHROUGH" %}
-        sources: {
-            {{source.to_feature_config()}}
-        }
-        {% endif %}
-        """)
-        key_list = ','.join(key for key in self.key_columns)
+        """)     
+        key_list = ','.join(key for key in self.features[0].key_alias) 
```

Comment:
I think we should keep the key provided at the `FeatureAnchor` level instead. It makes it explicit that all Features in the FeatureAnchor shares the same key

Otherwise, I think we should verify that the key is consistent across all features put in the same anchor and error otherwise.

I'm ok with both